addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -605,14 +605,16 @@ export class AnalyticDistribution extends Component {
     }
 
     onWindowClick(ev) {
-        //TODO: dragging the search more dialog should not close the popup either
-        const modal = document.querySelector('.modal:not(.o_inactive_modal)');
-        const clickedInSearchMoreDialog = modal && modal.querySelector('.o_list_view') && modal.contains(ev.target);
-        const clickedInKanbanSelectorDialog = modal && modal.querySelector('.o_kanban_view') && modal.contains(ev.target);
+        //FIX: Prevent editor closing when using command palette
```

Comment:
I don't think we need this comment, maybe replace it with all cases when the popup should not close